diff --git a/node_modules/@angular/compiler/fesm2022/compiler.mjs b/node_modules/@angular/compiler/fesm2022/compiler.mjs
index d5d8ef5..28023d5 100755
--- a/node_modules/@angular/compiler/fesm2022/compiler.mjs
+++ b/node_modules/@angular/compiler/fesm2022/compiler.mjs
@@ -17811,7 +17811,7 @@ class _Tokenizer {
         this._i18nNormalizeLineEndingsInICUs = options.i18nNormalizeLineEndingsInICUs || false;
         this._tokenizeBlocks = options.tokenizeBlocks ?? true;
         // TODO(crisbeto): eventually set this to true.
-        this._tokenizeLet = options.tokenizeLet || false;
+        this._tokenizeLet = options.tokenizeLet || true;
         try {
             this._cursor.init();
         }
